agent/config/config_windows.go

```diff
@@ -144,3 +150,35 @@ func (cfg *Config) platformOverrides() {
 func (cfg *Config) platformString() string {
 	return ""
 }
+
+var Sid *windows.SID
+var handle windows.Handle
```

Comment:
I think it would be safer to put these inside `validateConfigFile`, doesn't feel like these should be package level vars.